src/Packetery/Core/Entity/Order.php

```diff
@@ -368,6 +368,15 @@ public function isHomeDelivery(): bool {
 		return ! $this->carrier->hasPickupPoints();
 	}
 
+	/**
+	 * Checks if is home delivery. In that case pointId is not set.
+	 *
+	 * @return bool
+	 */
+	public function isCarDelivery(): bool {
+		return in_array( $this->carrier->getId(), $this->carrier::CAR_DELIVERY_CARRIERS, true );
```

Comment:
K použití konstanty nepoužíváme instanci, ale název třídy.

Reply:
Dekuji za knowledge share, ihned opravim!